governance/program/src/instruction.rs

```diff
@@ -492,8 +494,13 @@ pub fn create_proposal(
     name: String,
     description_link: String,
     governing_token_mint: &Pubkey,
+    proposal_index: u16,
```

Comment:
nit: We can have the total votes as a `u16` no problem, but I'm wondering if there may be many scratch proposals created before one is actually put to a vote.  Take it or leave it, probably all good

Reply:
I'm not sure, so I'll bump it in the next PR